Jobs/GenerateAutoDraft.php

```
@@ -0,0 +1,216 @@
+     * @param AISettings $settings
+     */
+    protected function storeDraft(Thread $thread, string $draft, AISettings $settings): void
+    {
+        // Store draft in thread's chatgpt column (same as manual generations)
+        $answers = $thread->chatgpt ? json_decode($thread->chatgpt, true) : [];
+        if ($answers === null) {
+            $answers = [];
+        }
+
+        // Mark as auto-draft
+        $draftData = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$draftData'. (undefined)

(UnusedLocalVariable)